ui/tract-renderer/src/main/java/org/cru/godtools/tract/viewmodel/CardViewHolder.java

```diff
@@ -114,6 +131,75 @@ private void bindLabel() {
         mDivider.setBackgroundColor(Styles.getTextColor(mModel));
     }
 
+    private void bindCardNavigation() {
+        int cardPositionCount = mModel != null ? mModel.getPosition() + 1 : 1;
+        Locale locale = mModel != null ? mModel.getManifest().getLocale() : Locale.getDefault();
+        int cardCount = getCardCount();
+        String positionText = String.format(locale, "%d/%d", cardPositionCount, cardCount);
+        mCardPositionView.setText(positionText);
+        if (isPrayerForm(mModel) || isPrayerSelection(mModel)) {
+            mPreviousCardView.setVisibility(View.INVISIBLE);
+            mPreviousCardView.setEnabled(false);
+            mCardPositionView.setVisibility(View.INVISIBLE);
+            mNextCardView.setVisibility(View.INVISIBLE);
+            mNextCardView.setEnabled(false);
+            return;
+        }
+        Context localContext = LocaleUtils.localizeContextIfPossible(mCardView.getContext(), locale);
```

Comment:
did Ann specifically ask to render the labels in the tool language and not the platform language?

Reply:
That was my understanding but I will verify.

Reply:
My gut is to leave the navigation labels in the device language since it's meant to help the owner of the device navigate. If I'm presenting it to someone in their native language I may not know what next and previous in their language is